email-utils.js

```diff
@@ -42,6 +43,9 @@ const EmailUtils = {
       return Promise.reject("SMTP transport not initialized");
     }
 
+    // Always include SERVER_URL and unsubscribeUrl for footer
+    aContext = Object.assign({"SERVER_URL": AppConstants.SERVER_URL}, aContext);
```

Comment:
Would this be easier if we just added the one property, instead of reassigning the entire instance?

```js
aContext.SERVER_URL = AppConstants.SERVER_URL;
```